### PR TITLE
typo fix and making some necessary modifications

### PR DESCRIPTION
**Pull request by Aditya-dom**
## Description

Fix #(models)

## Type of change
- typo-fix
- bug-fix
Please check the option that is related to your PR.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - In this case, we recommend to discuss your modification on GitHub issues before creating the PR
- [ ] Documentations (modification for documents)

## Checklist

- [x] My code follows [the style guidelines of this project](https://deepchem.readthedocs.io/en/latest/development_guide/coding.html)
  - [x] Run `yapf -i <modified file>` and check no errors (**yapf version must be  0.32.0**)
  - [x] Run `mypy -p deepchem` and check no errors
  - [x] Run `flake8 <modified file> --count` and check no errors
  - [x] Run `python -m doctest <modified file>` and check no errors
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: deepchem/models/keras_model.py [hunk 1]
```diff
@@ -19,6 +19,15 @@
...
 try:
+    from deepchem.data import Dataset, NumpyDataset
+    from deepchem.metrics import Metric
+    from deepchem.models.losses import Loss
+    from deepchem.models.models import Model
+    from deepchem.models.optimizers import Adam, Optimizer, LearningRateSchedule
+    from deepchem.trans import Transformer, undo_transforms
+    from deepchem.utils.evaluate import GeneratorEvaluator
+    from deepchem.utils.typing import LossFn, OneOrMany
+    from deepchem.models.wandblogger import WandbLogger
```
Comment (shreyasvinaya) [inline deepchem/models/keras_model.py:30]: these are duplicate imports, please remove them

Comment (Aditya-dom) [reply deepchem/models/keras_model.py:30]: okay

**Comment by shreyasvinaya** [comment]
@Aditya-dom Can you please explain the changes that you have made in your PR description

@rbharath could you please turn on the tests for this PR

**Comment by Aditya-dom** [comment]
> @Aditya-dom Can you please explain the changes that you have made in your PR description

> @shreyasvinaya yes sir sure, I can explain the changes that I have made.

> In the build method, changed `self.alphas = tf.Variable(...) to self.alphas = self.add_weight(...)`. This is a more standard way of adding trainable weights in Keras layers.

> In the call method, replaced get_shape() with shape when accessing the shape of the input tensor (inputs[0].shape[-1]). get_shape() is an older method, and using shape is recommended for compatibility.

> Simplified the calculation of subspace_size by using integer division (// 2) instead of int(... / 2) for better readability.

> Changed the loop in the call method to iterate over the range (0, len(subspaces), 2) instead of (row, row + 2) for clarity.

> Simplified the concatenation in the call method using tf.concat by directly providing the list of tensors to concatenate and  the axis along which to concatenate. 

> Thank you @shreyasvinaya  Sir for reviewing  my PR request.

**Review by rbharath — COMMENTED** [review]
The TensorFlow part of the codebase is in maintenance only (we are focusing on PyTorch porting as the longterm standard)

**Comment by Aditya-dom** [comment]
> The TensorFlow part of the codebase is in maintenance only (we are focusing on PyTorch porting as the longterm standard)

Okay Sir, Thank you

**Comment by Aditya-dom** [comment]
> The TensorFlow part of the codebase is in maintenance only (we are focusing on PyTorch porting as the longterm standard)

@rbharath sir, I am closing the PR. will you please confirm if I can proceed?
